nemo_text_processing/text_normalization/sv/taggers/tokenize_and_classify_lm.py

```diff
@@ -0,0 +1,201 @@
+            telephone_graph = TelephoneFst(deterministic=True).fst
+            electronic_graph = ElectronicFst(deterministic=True).fst
+            money_graph = MoneyFst(cardinal=cardinal, decimal=decimal, deterministic=True).fst
+            whitelist = WhiteListFst(input_case=input_case, deterministic=False, input_file=whitelist)
+            whitelist_graph = whitelist.graph
+
+            v_cardinal = vCardinalFst(deterministic=True)
+            v_cardinal_graph = v_cardinal.fst
+            v_decimal = vDecimalFst(deterministic=True)
+            v_decimal_graph = v_decimal.fst
+            v_ordinal = vOrdinalFst(deterministic=True)
+            v_ordinal_graph = v_ordinal.fst
+            v_fraction = vFractionFst(deterministic=True)
+            v_fraction_graph = v_fraction.fst
+            v_telephone_graph = vTelephoneFst(deterministic=True).fst
+            v_electronic_graph = vElectronicFst(deterministic=True).fst
+            v_measure = vMeasureFst(decimal=decimal, cardinal=cardinal, fraction=fraction, deterministic=False)
+            v_measure_graph = v_measure.fst
+            v_time_graph = vTimeFst(deterministic=True).fst
+            v_date_graph = vDateFst(deterministic=deterministic).fst
+            v_money_graph = vMoneyFst(decimal=decimal, deterministic=deterministic).fst
+            v_word_graph = vWordFst(deterministic=deterministic).fst
+
+            cardinal_or_date_final = plurals._priority_union(date_graph, cardinal_graph, NEMO_SIGMA)
+            cardinal_or_date_final = pynini.compose(cardinal_or_date_final, (v_cardinal_graph | v_date_graph))
+            time_final = pynini.compose(time_graph, v_time_graph)
+            ordinal_final = pynini.compose(ordinal_graph, v_ordinal_graph)
+            sem_w = 1
+            word_w = 100
+            punct_w = 2
+            classify_and_verbalize = (
+                pynutil.add_weight(time_final, sem_w)
+                | pynutil.add_weight(pynini.compose(decimal_graph, v_decimal_graph), sem_w)
+                | pynutil.add_weight(pynini.compose(measure_graph, v_measure_graph), sem_w)
+                | pynutil.add_weight(ordinal_final, sem_w)
+                | pynutil.add_weight(pynini.compose(telephone_graph, v_telephone_graph), sem_w)
+                | pynutil.add_weight(pynini.compose(electronic_graph, v_electronic_graph), sem_w)
+                | pynutil.add_weight(pynini.compose(fraction_graph, v_fraction_graph), sem_w)
+                | pynutil.add_weight(pynini.compose(money_graph, v_money_graph), sem_w)
+                | pynutil.add_weight(cardinal_or_date_final, sem_w)
+                | pynutil.add_weight(whitelist_graph, sem_w)
+                | pynutil.add_weight(v_word_graph, 1.1001)
+            ).optimize()
+
+            classify_and_verbalize = pynutil.insert("< ") + classify_and_verbalize + pynutil.insert(" >")
+            classify_and_verbalize |= pynutil.add_weight(word_graph, word_w)
+
```

Comment:
Is there a difference between this file and "tokenize_and_classify_with_audio"? For Eng, this file uses "lm" argument that alters graphs, e.g., limits the number of options for "lm=True".

Reply:
Not at the moment. I'll drop it until there's a need for it.